mailing_list/static/mailing_list/js/app.js

```diff
@@ -23,16 +23,29 @@
         ml.isUpdating = false;
         ml.primarySectionLists = [];
         ml.otherSectionLists = [];
+        ml.updatedAccessLevel = '';
```

Comment:
If you use the model directly, the underlying page also changes (the values underneath the modal, on the calling page, change as you change the modal). If there's a nicer way to use the model to build the option list and update the backend, but not update the underlying page bindings until the ajax call is successful, I could definitely change this. Perhaps it also merits a comment, since it isn't clear in the code.